lib/metanorma/m3aawg/processor.rb

```diff
@@ -25,9 +25,9 @@ def fonts_manifest
         {
           "Arial" => nil,
           "Cambria Math" => nil,
-          "Courier" => nil,
           "Courier New" => nil,
-          "EB Garamond" => nil,
+          "EB Garamond 08" => nil,
+          "EB Garamond 12" => nil,
           "Overpass" => nil,
```

Comment:
@Intelligent2013 Do we need "Overpass Light" and others, or it's enough to have just Overpass?

Reply:
The font 'Overpass' isn't using in mn2pdf. But I don't sure about html, doc...

Reply:
@opoudjis Please can you help us with the html/doc formats? May they need Overpass Light or some other forms of the Overpass font?

Reply:
Ping @opoudjis

Reply:
What is the question I am being pinged about? Is this which flavours Overpass is being used in?

Only Overpass is mentioned, not Overpass Light. Overpass is mentioned in metanorma-generic, OGC, M3AAWG, MPFA.

* @Intelligent2013 has done careful design of OGC PDF, so if he isn't using Overpass in PDF, that's because OGC didn't ask for it. OGC HTML does use Overpass. OGC DOC does not. That was likely our decision.
* I've got rid of the mention of Overpass in MPFA: it was left in as a Google Fonts link, but we've switched MPFA to Titillium. MPFA is in any case a theory-only flavour currently (and has been for years).
* M3AAWG uses Overpass in HTML.  That likely did not come from M3AAWG.

I doubt any of our rendering is so refined, that we make a distinction between normal weight and light weight of fonts. I would assume we don't need Overpass Light, and if we were really pushed for whatever reason, we could get rid of Overpass itself: I'm not seeing an indication that any SDO requested it.

Reply:
Thanks @opoudjis for this investigation!

I think this can be resolved then.